core/string/ustring.cpp

```diff
@@ -1124,6 +1124,26 @@ int String::get_slice_count(String p_splitter) const {
 	return slices;
 }
 
+int String::get_slice_count(const char *p_splitter) const {
+	if (is_empty()) {
+		return 0;
+	}
+	if (*p_splitter == '\0') {
```

Comment:
```suggestion
	if (p_splitter == nullptr || *p_splitter == '\0') {
```
Realized there's no check to handle null strings here